bot/admin/web/src/app/analytics/flow/flow.component.html

```diff
@@ -466,7 +466,7 @@
 <div *ngIf="selectedLayout === 'Sankey'">
   <nb-card>
     <nb-card-body>
-      <google-chart
+      <!-- <google-chart
```

Comment:
Can create an issue on tock repository to follow the change with  Apache Echarts  graphs and mention it (the issue number) as a comment here.

Reply:
https://github.com/theopenconversationkit/tock/issues/1714